soh/soh/OTRGlobals.cpp

```diff
@@ -1707,8 +1701,8 @@ extern "C" int CustomMessage_RetrieveIfExists(GlobalContext* globalCtx) {
             } else {
                 messageEntry = Randomizer_GetGanonHintText();
             }
-        } else if (textId == TEXT_SCRUB_POH || textId == TEXT_SCRUB_STICK_UPGRADE || textId == TEXT_SCRUB_NUT_UPGRADE) {
-            messageEntry = Randomizer_GetScrubMessage(textId);
+        } else if (textId >= 0x9000 || textId <= 0x905F) {
```

Comment:
This was my solution to getting around not having access to actorParams, this is also what 3drando uses for scrub messages.

Reply:
A couple of points here. 3ds and n64 text stuff doesn't necessarily match up from my experience, so we need to make sure those are actually the ones the game uses. This might be what's causing the bug Flax found with the recovery hearts, for instance.

Reply:
This should be `&&`, not `||`. With `||`, all textIds below `0x905F` were matching, and I think a few items were matching textIds in our Scrub Message Table.

Reply:
```suggestion
        } else if (textId >= 0x9000 && textId <= 0x905F) {
```